Lib/javascript/v8/javascriptrun.swg

```diff
@@ -67,6 +75,7 @@ typedef v8::PropertyCallbackInfo<v8::Value> SwigV8PropertyCallbackInfo;
 #define SWIGV8_NUMBER_NEW(num) v8::Number::New(num)
 #define SWIGV8_OBJECT_NEW() v8::Object::New()
 #define SWIGV8_UNDEFINED() v8::Undefined()
+#define SWIGV8_OBJECT v8::Handle<v8::Value>
```

Comment:
This seems like a very confusing name, knowing that we also use `v8::Handle<v8::Object>` in a couple of places. Why not call it `SWIGV8_VALUE` instead?

Reply:
I see your point and will rename it.